homeassistant/components/scene/fibaro.py

```diff
@@ -0,0 +1,55 @@
+"""
+Support for Fibaro scenes.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/scene.fibaro/
+"""
+import logging
+
+from homeassistant.components.scene import (
+    DOMAIN, Scene)
+from homeassistant.components.fibaro import (
+    FIBARO_CONTROLLER, FIBARO_DEVICES, FibaroDevice)
+
+ENTITY_ID_FORMAT = DOMAIN + '.{}'
+
+DEPENDENCIES = ['fibaro']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Perform the setup for Fibaro scenes."""
+    if discovery_info is None:
+        return
+
+    add_entities(
+        [FibaroScene(scene, hass.data[FIBARO_CONTROLLER])
+         for scene in hass.data[FIBARO_DEVICES]['scene']], True)
+
+
+class FibaroScene(FibaroDevice, Scene):
```

Comment:
Yes, it's a unique id and the parent class of this (FibaroDevice) is retuning it as unique_id property. Should I just leave the entity_id attribute unfilled? Would that cause a breaking change or will I just end up with the same result anyway?

Reply:
You can safely remove the filling of the entity ID for all Fibaro components.

 - For new entities, it was never there, so it's fine
 - For existing entities, it was used as a suggestion when registering the entity ID, it is no longer used.

The nice thing is that now Home Assistant will start generating entity IDs based on name instead of unique ID, making it a more pleasant experience for the user.

Reply:
Removed it :-)